src/api/app/models/kiwi/image/xml_builder.rb

```diff
@@ -44,6 +46,18 @@ def update_preferences(document)
         document
       end
 
+      def update_preference_version(document)
+        if @image.preference.version.present?
+          if document.xpath('image/preferences/version').any?
+            document.xpath('image/preferences/version').remove
```

Comment:
I would prefer to replace and not remove to keep the diff as small as possible. So you would need two different cases.

Reply:
@ChrisBr what do you mean "to keep the diff as small as possible"? The end result is the exact same?

Reply:
Only if the version is indeed the first child. If it is on a different position, you would change the position as you add it again as first child (see line 55).